### DISCUSSION TITLE
unable to send email invites

### DISCUSSION THREAD

**Discussion by ssriley** [bionic-gpt/bionic-gpt#227] [Q&A]
I have tried to send emails to other team members and the send fails. I have setup the env variables in the app service in the docker compose file. When i hit the send button this is what is logged in the app container:

Could not send email: lettre::transport::smtp::Error { kind: Network, source: Custom { kind: InvalidData, error: InvalidMessage(InvalidContentType) } }

The email server i am using is using self signed certs. Not sure if that is an issue. But it looks to be something else here.

**Comment by rpaGuyai** [comment]
Were you able to get this resolved? I am facing same issue and unable to find any resolution yet.https://github.com/bionic-gpt/bionic-gpt/issues/474
